modules/dnf/module.yml

```
@@ -0,0 +1,36 @@
+name: dnf
+shortdesc: The dnf module offers pseudo-declarative package and repository management using dnf.
+example: |
+  type: dnf
+  install-weak-dependencies: true
```

```suggestion
  type: dnf
```
This doesn't appear to be in the top-level spec.

---

Yes, it's a leftover when I wanted to add only this option, but then changed my mind & added more options, + didn't make it global like here.